_includes/pbc/all/install-features/202212.0/install-the-customer-account-management-feature.md

```diff
@@ -0,0 +1,1029 @@
+
+
+This document describes how to integrate the Customer Account Management feature into a Spryker project.
+{% info_block errorBox "Included features" %}
+
+The following feature integration guide expects the basic feature to be in place. The current feature integration guide only adds:
+* Redirect support for Customer login functionality.
+* Password set and reset console commands for customers.
+* Double opt-in for customer registration.
+* OAuth
+* Improved password security
+* Session validation
+
+{% endinfo_block %}
+
+## Install feature core
+
+Follow the steps below to install the Customer Account Management feature core.
+
+### Prerequisites
+
+Overview and install the necessary features before beginning the integration.
+
+| NAME         | VERSION          | INTEGRATION GUIDE                                                                                                                    |
+|--------------|------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core | {{site.version}} | [Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/spryker-core-feature-integration.html) |
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker-feature/customer-account-management: "{{site.version}}" spryker/oauth-customer-connector:"^1.6.0" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the following modules have been installed:
+
+| MODULE                 | EXPECTED DIRECTORY                      |
+|------------------------|-----------------------------------------|
```

Comment:
there are modules present at feature
https://release.spryker.com/features/view/54

also, SessionCustomerValidationPage and SessionCustomerValidationPageExtension will be mapped to the same feature on release

Reply:
Added missing modules. Modules SessionCustomerValidationPage and SessionCustomerValidationPageExtension are mentioned in a separate section `Install feature frontend` below (the similar approach as for the Agent Assist feature).